src/main/java/sirius/kernel/health/Average.java

```diff
@@ -73,6 +74,7 @@ public void addValue(double value) {
      */
     public void addValues(long numberOfValues, double sumOfValue) {
         long newCount = count.addAndGet(numberOfValues);
+        totalCount.addAndGet(numberOfValues);
```

Comment:
what is the overflow behaviour - should be somehow reset it before Long.MAX_VALUE is reached?